network-api/networkapi/templates/partials/banner.html

```diff
@@ -35,7 +35,9 @@
                 </div>
                 <div class="tw-row">
                     <div class="tw-w-full tw-px-4">
-                        <a class="btn btn-banner tw-w-full" href="https://donate.mozilla.org/?utm_source=foundation.mozilla.org&utm_medium=referral&utm_campaign=fmonav&utm_content=header" target="_blank" rel="noopener noreferrer">{% trans "Support Mozilla" %}</a>
+                        <a class="btn btn-banner tw-w-full" href="https://donate.mozilla.org?utm_source=foundation.mozilla.org&utm_medium=referral&utm_campaign=fmonav&utm_content=2021_EOY_bannner_MoFo" target="_blank" rel="noopener noreferrer">
```

Comment:
the CMS will inject a trailing `/` if it doesn't exist, so if we keep it in here, it won't first redirect.
```suggestion
                        <a class="btn btn-banner tw-w-full" href="https://donate.mozilla.org/?utm_source=foundation.mozilla.org&utm_medium=referral&utm_campaign=fmonav&utm_content=2021_EOY_bannner_MoFo" target="_blank" rel="noopener noreferrer">
```